Justfile

```diff
@@ -51,6 +51,8 @@ build +tools: build-protos build-zips build-frontend
   #!/bin/bash
   shopt -s extglob
 
+  git diff
+
```

Comment:
Is this for logging @gak?

Reply:
Yes, I need to see what is causing the repo to be dirty.